### PR TITLE
[clang][Interp] Implement builtin_expect

### PR DESCRIPTION
**Pull request by tbaederr**

### CODE REVIEW

**Comment by tbaederr** [comment]
Ping

**Comment by tbaederr** [comment]
Ping

**Comment by tbaederr** [comment]
Ping

**Comment by tbaederr** [comment]
Ping

**Comment by tbaederr** [comment]
Ping

**Review by cor3ntin — APPROVED** [review]
LGTM

File: clang/lib/AST/Interp/InterpBuiltin.cpp [hunk 1]
```diff
@@ -34,6 +34,19 @@ PrimType getIntPrimType(const InterpState &S) {
   llvm_unreachable("Int isn't 16 or 32 bit?");
 }
 
+PrimType getLongPrimType(const InterpState &S) {
```
Comment (cor3ntin) [inline clang/lib/AST/Interp/InterpBuiltin.cpp:37]: I don't suppose we can merge with getIntPrimType ?

**Pull request merged**